File: Codezilla-2 Solns/ModifiedFizzBuzz.cpp
#include <iostream>
#include <string>
#include <algorithm>

bool charPresent(std::string S, char ch)
{
    return find(S.begin(),S.end(),ch) != S.end();
}

int main() 
{
    int n,flag;
    std::string str;
    std::cin >> n;
    for(int i=1;i<=n;i++)
    {
        flag = 0;
        str = std::to_string(i);
        if(charPresent(str,'3'))
        {
            std::cout << "FIZZ";
            flag++;
        }
        if(charPresent(str,'5'))
        {
            std::cout << "BUZZ";
            flag++;
        }
        if(!flag)
        {
            std::cout << str;
        }
        std::cout << std::endl;
    }
    return 0;
}

File: Codezilla-2 Solns/ModifiedFizzBuzz.py
n = int(input())

for i in range(1,n+1):
    s = str(i)
    if "3" not in s and "5" not in s:
        print(i)
        continue
    if "3" in s:
        print("FIZZ",end="")
    if "5" in s:
        print("BUZZ",end="")
    print()